### ISSUE TITLE
ng6-cli Logo

### ISSUE THREAD

**Issue by BenDiuguid** [UltimateSoftware/ng6-cli#33]
We need a Logo!

**Comment by asleepinglion** [comment]
With our likely move to Angular 2+ and the semantic versioning of Angular 2+, NG6 (which originally stood for NG + ES6) will be kind of ambiguous when Angular 6 releases in March 2018. Perhaps we should think about changing the name to be more future proof, similar to how 6to5 became babel. :)

**Comment by BenDiuguid** [comment]
Yes I totally agree with this, also it's not super clear that NG6 stood for NG + ES6 (as in I had forgotten 😆 )

also this can easily use more than ES6, so I support the name change! Not sure what it should be though?
